lib/ansible/modules/cloud/misc/proxmox.py

```diff
@@ -175,6 +175,14 @@
      - Indicate desired state of the instance
     choices: ['present', 'started', 'absent', 'stopped', 'restarted']
     default: present
+  pubkey:
+    version_added: "2.3"
+    description:
+      - Public key to add to /root/.ssh/authorized_keys
```

Comment:
Apologies, my comment wasn't clear.
I assume this module could be ran as root *or* another user. Therefore the `description:` should be something closer to
`Public key to add to users .ssh/authorized_keys.